Portage/GHCCore.hs

```diff
@@ -144,6 +148,40 @@ ghc741 = (ghc [7,4,1], mkIndex ghc741_pkgs)
 -- Source: http://haskell.org/haskellwiki/Libraries_released_with_GHC
 --         and our binary tarballs (package.conf.d.initial subdir)
 
+
+ghc821_pkgs :: [Cabal.PackageIdentifier]
+ghc821_pkgs =
+  [ p "array" [0,5,2,0]
+  , p "base" [4,10,0,0]
+  , p "binary" [0,8,5,1] -- used by libghc
+  , p "bytestring" [0,10,8,2]
+--  , p "Cabal" [2,0,0,0]  package is upgradeable
+  , p "containers" [0,5,10,2]
+  , p "deepseq" [1,4,3,0] -- used by time
+  , p "directory" [1,3,0,2]
+  , p "filepath" [1,4,1,2]
+  , p "ghc-boot" [8,2,1]
+--  , p "ghc-boot-th" [] err... vOv
+  , p "ghc-compact" [0,1,0,0] -- new addition
+  , p "ghc-prim" [0,5,1,0]
+  , p "ghci" [8,2,1]
+  , p "hoopl" [3,10,2,2] -- used by libghc
+  , p "hpc" [0,6,0,3] -- used by libghc
+  , p "integer-gmp" [1,0,0,1]
+--  , p "pretty" [] the copy in ghc is used only by ghc
+--  https://hackage.haskell.org/package/ghc-8.2.1/docs/src/Pretty.html
+  , p "process" [1,6,1,0]
+  , p "template-haskell" [2,12,0,0] -- used by libghc
+  , p "time" [1,8,0,1] -- used by unix, directory, hpc, ghc. unsafe to upgrade
+-- transformers is not actually part of ghc and infact a separate package, could
+-- this be removed?
+  , p "transformers" [0,5,4,0] -- used by libghc
+  , p "unix" [2,7,2,2]
+-- could this be added (because Prefix and WSL)?
+--  , p "Win32" [2,5,4,1]
```

Comment:
"Win32" should not be normally visible when dependency resolution happens.
I don't think Win32 package works on WSL.